api/src/documents.py

```diff
@@ -32,14 +29,13 @@
 
 from rq import Queue
 from redis import Redis
-from rq.exceptions import NoSuchJobError
-from rq import job
-
-from src.embedder_engine import embedder
+# from src.embedder_engine import embedder
+from pydantic import BaseModel
+from openai.embeddings_utils import get_embeddings
 from src.semantic_highlighter import highlighter
 
-from pydantic import BaseModel
 
+PARAGRAPHS_INDEX = "paragraphs"
```

Comment:
The only change required for this to not be a DRAFT PR is to change the paragraph index back to `document_paragraphs` once we set up the alises in es/prod.